docs/running/wdl.rst

```diff
@@ -0,0 +1,139 @@
+WDL Support in Toil
+********
+
+Support is still in the alpha phase and should be able to handle basic wdl files.  See the specification below for more
+details.
+
+How to Run a WDL file in Toil
+-----------
+Recommended best practice when running wdl files is to first use the Broad's wdltool for syntax validation and generating the
+needed json input file.  Full documentation can be found on the repository_, and a precompiled jar binary can be downloaded here: wdltool_.
+
+That means two steps.  First, make sure your wdl file is valid and devoid of syntax errors by running:
+
+``java -jar wdltool.jar validate example_wdlfile.wdl``
+
+Second, generate a complementary json file if your wdl file needs one.  This json will contain keys for every necessary
+input that your wdl file needs to run:
+
+``java -jar wdltool.jar inputs example_wdlfile.wdl``
+
+When this json template is generated, open the file, and fill in values as necessary by hand.  WDL files all require
+json files to accompany them.  If no variable inputs are needed, a json file containing only '{}' may be required.
+
+Once a wdl file is validated and has an appropriate json file, workflows can be run in toil using:
+
+``toil-wdl-runner example_wdlfile.wdl jsonfile.json``
+
+See options below for more parameters.
+
+ENCODE Example from ENCODE-DCC
+-----------
+To follow this example, you will need docker installed.  The original workflow can be found here:
+https://github.com/ENCODE-DCC/pipeline-container
+
+We've included the wdl file and data files in the toil repository needed to run this example.  First, find the wdl file:
+'src/toil/test/wdl/wdl_templates/testENCODE/encode_mapping_workflow.wdl'
```

Comment:
It would be easier for the user to just link the downloadable file as is done for the sort example

``Download :download:`the example code <../../src/toil/test/sort/sort.py>`.``

Reply:
Amended, though it seems it'll need to be merged before I can see it properly on the docs?